### ISSUE TITLE
Alert metadata lookup not working as expected

### ISSUE THREAD

**Issue by severin** [dependabot/fetch-metadata#464]
Hi there!

I'm trying to implement a workflow that labels pull requests that address dependabot alerts. To identify these PRs my idea was to use the *fetch-metadata* action and look at the `alert-state`: if it is `OPEN` then the PR addresses an open dependabot alert.

However, it seems like the *fetch-metadata* is not working correctly (or maybe I'm just misunderstanding it). 

### Observed behaviour

- Dependabot opened a PR that bumps the *minimatch* dependency from 3.0.4 to 3.1.2:
<img width="902" alt="Screenshot 2023-10-30 at 11 41 45" src="https://github.com/dependabot/fetch-metadata/assets/3077/857442b3-22bb-4947-9755-4369e3e8bd10">

- This PR addresses a security alert:
<img width="1193" alt="Screenshot 2023-10-30 at 11 42 07" src="https://github.com/dependabot/fetch-metadata/assets/3077/240ceb44-20bc-43bb-a390-2dc8e3e17717">

- Running `fetch-metadata` on the PR does not return any alert information (even though `alert-lookup` is set to `true`):

<img width="569" alt="Screenshot 2023-10-30 at 11 42 39" src="https://github.com/dependabot/fetch-metadata/assets/3077/b0952c02-feb0-4aa1-9e43-f49af83605e3">

### Expected behaviour

I expected the `fetch-metadata` action to output alert information:

```
outputs.alertState: OPEN
outputs.ghsa-id: GHSA-f8q6-p94x-37v3
outputs.cvss: 7.5
```

Am I doing something wrong or is this a bug?

**Comment by Nishnha** [comment]
Hi @severin you'll need to use a PAT with access to security alerts in order to populate that info.

We have this blurb about this under `github-token` in the readme, but maybe we should also put it under `alert-lookup` and `compat-lookup`.

If you were using a token with the right permissions then lmk and I'll look into this more closely.

**Comment by jeffwidman** [comment]
Can this be closed now that #466 is merged?
